src/engine/sparqlExpressions/AggregateExpression.h

```diff
@@ -104,7 +104,10 @@ using AGG_EXP = AggregateExpression<
 // with arguments and result of type `NumericValue` (which is a `std::variant`).
 template <typename NumericOperation>
 inline auto makeNumericExpressionForAggregate() {
-  return [](const std::same_as<NumericValue> auto&... args) -> NumericValue {
+  return [](const auto&... args)
+             -> CPP_ret(NumericValue)(
```

Comment:
with a `<typename... Args>` you get rid of the `decay_t<decltype>` , which is ugly.